app/tests/lectures/test_urls.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from django.test import Client
+from rest_framework.reverse import reverse
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("url", [reverse("lecture-list-list")])
+def test_lecture_list_response(url):
+    """
+    Ensure we can connect to lecture list url
+    """
+    client = Client()
+    assert 200 == client.get(url).status_code
```

Comment:
Please compare status codes with `status` object
`from rest_framework import status`, not with an integer.